api-report/container-loader.api.md

```diff
@@ -283,6 +283,7 @@ export interface IDeltaManagerInternalEvents extends IDeltaManagerEvents {
 // @public
 export type IDetachedBlobStorage = Pick<IDocumentStorageService, "createBlob" | "readBlob"> & {
     size: number;
+    all(): string[];
```

Comment:
The shape does not look right, assuming 400Mb of payload I've heard being quoted for these kinds of workloads.
I think it has to be enumeration and it has to be async. That might create other problems (async part of it), but I'm not sure duplicating that much memory is scalable solution.

Reply:
Oh, these are IDs? Sorry, misunderstood, I'd change the name and add comment.